Bindings/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+
+if(BUILD_PYTHON_WRAPPING)
+    find_package(SWIG 3.0.5 REQUIRED)
+    add_subdirectory(Python)
+endif()
+if(BUILD_PYTHON_WRAPPING)
+    find_package(SWIG 3.0.5 REQUIRED)
+    add_subdirectory(Java)
+endif()
```

Comment:
That's possible though I can imagine different languages requiring different swig versions or Python wrapping requiring other packages (e.g. numpy, scipy, ..) that will be added separately. Is it worth the tradeoff to avoid duplicating the find_package line in CMake? I'm open.